src/main/java/se/sundsvall/smloader/integration/db/model/CaseEntity.java

```diff
@@ -5,25 +5,40 @@
 import jakarta.persistence.EnumType;
 import jakarta.persistence.Enumerated;
 import jakarta.persistence.Id;
-import jakarta.persistence.Lob;
 import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
 import org.hibernate.Length;
+import org.hibernate.annotations.UuidGenerator;
 
-import java.sql.Clob;
 import java.util.Objects;
 
 @Entity
-@Table(name = "'case'")
+@Table(name = "'case'",
+	uniqueConstraints = {
+		@UniqueConstraint(name = "uq_open-e_case_id_instance", columnNames = {"open-e-case-id", "instance"})
+	})
 public class CaseEntity {
 
 	@Id
+	@UuidGenerator
+	@Column(name = "id")
 	private String id;
 
-	@Lob
-	@Column(length = Length.LONG32)
-	private Clob openECase;
+	@Column(name = "family_id")
+	private String familyId;
 
-	@Column(columnDefinition = "varchar(255)")
+	@Column(name = "open-e-case-id")
+	private String openECaseId;
+
+	@Column(name = "instance")
+	@Enumerated(EnumType.STRING)
+	private Instance instance;
+
+
+	@Column(name = "open-e-case", length = Length.LONG32)
```

Comment:
externalCaseId?

If you keep the name, use same naming convention as the other column names (underscore instead of hyphens).

The same goes for "open-e-case"